lib/elixir/test/elixir/keyword_test.exs

```diff
@@ -42,4 +42,35 @@ defmodule KeywordTest do
       Keyword.get_and_update!([a: 1], :a, fn value -> value end)
     end
   end
+
+  test "merge/2" do
```

Comment:
it is not. you only have 3 doctests, while those three repeat, I doubt you can test every single case in here.